app/controllers/foreman_salt/minions_controller.rb

```diff
@@ -6,14 +6,21 @@ class MinionsController < ::ForemanSalt::ApplicationController
     add_smart_proxy_filters :node, :features => 'Salt'
 
     def node
-      enc = {}
+      enc = {'foreman' => {}}
```

Comment:
BTW We can close a pull request and I can create another one that just render the yaml if string starts with '---'